azuredevops/internal/acceptancetests/resource_serviceendpoint_github_enterprise_test.go

```diff
@@ -0,0 +1,155 @@
+// +build all resource_serviceendpoint_github_enterprise
+// +build !exclude_serviceendpoints
+
+package acceptancetests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+)
+
+func TestAccServiceEndpointGitHubEnterprise_PersonalTokenBasic(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	serviceEndpointName := testutils.GenerateResourceName()
+
+	resourceType := "azuredevops_serviceendpoint_github_enterprise"
+	tfSvcEpNode := resourceType + ".serviceendpoint"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, &[]string{"AZDO_GITHUB_ENTERPRISE_SERVICE_CONNECTION_PAT"}) },
```

Comment:
`personal_access_token` can be set with dummy values,  remove `AZDO_GITHUB_ENTERPRISE_SERVICE_CONNECTION_PAT` from the the precheck,  so we do not need to update the CI pipeline when new environment variables add to the provider.